lib/gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -87,7 +87,6 @@ def self.raw_connect(username, password, auth_url, service = "Compute", extra_op
       $fog_log.warn("MIQ(#{self.class.name}##{__method__}) "\
                     "Service #{service} not available for openstack provider #{auth_url}")
       $fog_log.warn(err.message)
-      $fog_log.warn(err.backtrace.join("\n"))
```

Comment:
Even though it's only one line, you're removing lots of unhelpful and confusing lines of backtraces  🔥 ✂️ 

👍